autogl/datasets/_heterogeneous_datasets/_dgl_heterogeneous_datasets.py

```diff
@@ -0,0 +1,132 @@
+import os
+import dgl.data.utils
+import numpy as np
+import scipy.io
+import torch
+from autogl.data import InMemoryStaticGraphSet
+from .. import _dataset_registry
+
+
+def get_binary_mask(total_size, indices):
+    mask = torch.zeros(total_size)
+    mask[indices] = 1
+    return mask.byte()
+
+
+@_dataset_registry.DatasetUniversalRegistry.register_dataset("hetero-acm-han")
```

Comment:
Shall we bind the dataset name with the name of model? @BeiniXie @ZW-ZHANG @CoreLeader 
I suggest check data source from other library.